### ISSUE TITLE
Find and Fix Bugs

### ISSUE THREAD

**Issue by nathfreder** [nathfreder/simple-react-boilerplate#15]
Hello,
I have noticed some bugs in the application and some things that could be improved. I thought it would be cool to create this issue and help some people out with Hacktoberfest. If you find a bug or something to improve make a PR! 

Thanks so much.

**Comment by shreyks** [comment]
Is this issue solved yet?

**Comment by nathfreder** [comment]
No, it is just an invitation for people to have a look at the source and help me report bugs and get a tee-shirt out of it.

**Comment by shreyks** [comment]
I new to this, could you tell me what bugs are you expecting, because it works fine for me

**Comment by nathfreder** [comment]
I am just looking for simple improvements or fixes, for instance, you could make the home page say something more interesting than 'Home!' or add a Cat GIF to the bottom of the page.

**Comment by shreyks** [comment]
Improve the page, add a nav-bar and stuff to it, work like that?

**Comment by nathfreder** [comment]
Yeah! That works!
